python/mlx/nn/layers/activations.py

```diff
@@ -18,7 +18,8 @@ def decorator(klass):
 
 @partial(mx.compile, shapeless=True)
 def sigmoid(x):
-    r"""Applies the sigmoid function.
```

Comment:
I know it looks better when indented but I think that having it be in the same line is recommended. Is it causing any problem anywhere?

Reply:
Yes 😞  it causes an issue in the interpretation of the docstring (it treats the first line on a different indentation level than the remainder which messes up how it renders. An alternative is to insert the proper amount of indentation in the first line so it matches the others before setting it on the wrapped function.

Reply:
I wanted to avoid dealing with a bunch of edge cases there.. if we consistently do not indentat the first line then it might be simple to patch it in the code itself.

Reply:
I added the indentation to the first line